README.md

```diff
@@ -66,43 +66,43 @@ T.get('account/verify_credentials', { skip_status: true })
 //
 //  retweet a tweet with id '343360866131001345'
 //
-T.post('statuses/retweet/:id', { id: '343360866131001345' }, function (err, data, response) {
+twit.post('statuses/retweet/:id', { id: '343360866131001345' }, (err, data, response) => {
   console.log(data)
 })
 
 //
 //  destroy a tweet with id '343360866131001345'
 //
-T.post('statuses/destroy/:id', { id: '343360866131001345' }, function (err, data, response) {
+twit.post('statuses/destroy/:id', { id: '343360866131001345' }, (err, data, response) => {
   console.log(data)
 })
 
 //
 // get `funny` twitter users
 //
-T.get('users/suggestions/:slug', { slug: 'funny' }, function (err, data, response) {
+twit.get('users/suggestions/:slug', { slug: 'funny' }, (err, data, response) => {
   console.log(data)
 })
 
 //
 // post a tweet with media
 //
-var b64content = fs.readFileSync('/path/to/img', { encoding: 'base64' })
+let b64contentwit = fs.readFileSync('/path/to/img', { encoding: 'base64' })
```

Comment:
this `let` and all of the `let`s below can be changed to `const`, since we don't edit these variables

Reply:
Great, on it.